src/main.js

```diff
@@ -22,5 +16,26 @@ document.MODES = {
     description: 'what starship from Star Wars is showed in the picture.',
   },
 };
-document.mode = 'characters';
-document.querySelector('.quiz-main-menu').addEventListener('click', changeGameMode);
+
+const indexPage = new IndexPageModes();
+
+const mainPhoto = new MainPhoto();
+document.getElementById('main-grid-container').appendChild(mainPhoto.render());
+
+const buttonsContainer = new ButtonsContainer();
+
+document.getElementById('main-grid-container').appendChild(indexPage.render()).appendChild(buttonsContainer.render());
+const splitted = document.getElementById('main-photo').src.split('/');
+document.mode = splitted[splitted.length - 2];
+const modeDesc = document.MODES[document.mode].header;
+document.querySelector('h2.main__header').innerHTML = modeDesc;
+
+document.addListenerToButtons = addListenerToButtons(
```

Comment:
There is an error related to this in the console: `Uncaught ReferenceError: addListenerToButtons is not defined`.